src/styles/_nodes.scss

```diff
@@ -35,6 +35,8 @@
 .iconAndLabel {
   flex-grow: 1;
   text-align: center;
+  display: flex;
+  justify-content: center;
 }
```

Comment:
Add `align-items: center;` to `.iconAndLabel { ... }` and replace the `.statusIcon` styles with:

```css
.statusIcon {
  margin: 0;
  padding-left: 20px;
}
```